charts/chart_deps/grafana/datasources/templates/datasources-lokiTpl.yaml

```diff
@@ -0,0 +1,39 @@
+{{- $ds := .Values.datasources.lokiTpl }}
```

Comment:
why do we need to introduce new template file. We can achieve the same in one datastources.yaml no?

Reply:
please explain what is lokiTpl why it is needed?